raster-foundry/raster-foundry
# Fix homepage links, scene preview modal init

## Overview

Fix broken home page tool links
Fix scene preview modal only zooming to scene on first preview

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Demo
![image](https://user-images.githubusercontent.com/4392704/32907930-ee1276be-cacf-11e7-87c3-0ca7f880f420.png)
![image](https://user-images.githubusercontent.com/4392704/32907946-f892718e-cacf-11e7-8557-e593219ce09c.png)


## Testing Instructions

 * Verify that the home page tool links now all work as expected
* Verify that all scene previews after the first zoom into the relevant scene, instead of doing nothing.

Closes #2713 
Closes #2709
